packages/kbn-apm-synthtrace/src/scenarios/infra_hosts_with_apm_hosts.ts

```diff
@@ -6,29 +6,30 @@
  * Side Public License, v 1.
  */
 import { InfraDocument, apm, Instance, infra, ApmFields } from '@kbn/apm-synthtrace-client';
+import { random, times } from 'lodash';
 import { Scenario } from '../cli/scenario';
 import { withClient } from '../lib/utils/with_client';
 import { getSynthtraceEnvironment } from '../lib/utils/get_synthtrace_environment';
 
 const ENVIRONMENT = getSynthtraceEnvironment(__filename);
 
-const scenario: Scenario<InfraDocument | ApmFields> = async (runOptions) => {
+const scenario: Scenario<InfraDocument | ApmFields> = async ({
+  logger,
+  scenarioOpts = { numInstances: 10 },
+}) => {
   return {
     generate: ({ range, clients: { infraEsClient, apmEsClient } }) => {
-      const { numServices = 3, numHosts = 10 } = runOptions.scenarioOpts || {};
-      const { logger } = runOptions;
+      const { numInstances } = scenarioOpts;
+      const transactionName = '240rpm/75% 1000ms';
```

Comment:
I noticed that the value `240rpm/75% 1000ms` is also reused in other Kibana files as transactionName.

This is completely outside the scope of this PR, but it might be valuable to suggest unifying these kind of values into a utilities package (though I'm not sure if "package" is the right term in this context).

Reply:
I actually need to change this transaction name. It could be a utility file, at least. But let's talk about this for future improvements.